code/game/objects/items/miscellaneous.dm

```diff
@@ -19,6 +19,62 @@
 	icon_state = "skub"
 	w_class = WEIGHT_CLASS_BULKY
 	attack_verb = list("skubbed")
+	
```

Comment:
Wedging it in between Skub and Skub's suicide is maybe not the best code organization.

Reply:
Difficulty with the merger in a conflict did that, i could simply leave scub suicide where it is and move scub's entry down instead so they can be together. Give me a moment.